<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWidget</class>
 <widget class="QWidget" name="StatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="statsLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="statsTitleLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Statistics Summary</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalTestsText">
         <property name="text">
          <string>Total Tests:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalTestsLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalWordsText">
         <property name="text">
          <string>Total Words Tested:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalWordsLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cumulativeText">
         <property name="text">
          <string>Cumulative Accuracy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cumulativeLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.0%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="highestText">
         <property name="text">
          <string>Highest Accuracy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="highestLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.0%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ResetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="chartLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
